src/observers/slack_resource.cpp

```diff
@@ -23,39 +24,59 @@ Try<Nothing> SlackResourceObserver::consume(const ResourceUsage& usage) {
 
       auto previousSample = this->previousSamples->find(executor);
       if (previousSample != this->previousSamples->end()) {
-        Result<double_t> slackResource = CalculateCpuSlack(
-            (*previousSample), executor);
-        if (slackResource.isSome()) {
-            LOG(INFO) << NAME << "Estimated revocable resources for "
-                      << executor.executor_info().name() << " FrameworkID("
-                      << executor.executor_info().framework_id() << "): "
-                      << slackResource.get();
-            cpuSlack += slackResource.get();
-        } else if (slackResource.isError()) {
-          LOG(ERROR) << slackResource.error();
+        Result<double_t> executorCpuSlack = CalculateCpuSlack(
+            *previousSample, executor);
+        Try<double_t> executorCpuUsage = CountCpuUsage(
+            *previousSample, executor);
+        if (executorCpuSlack.isSome()) {
+          slackResources += executorCpuSlack.get();
```

Comment:
Yes. I'm not sure we need to spam logs with this amount of detail like 'estimated slack for each executor'.
I would rather log "estimated X slack' for whole pipeline. 
What do you think?